packages/babel/src/index.mjs

```diff
@@ -91,16 +91,16 @@ export default function (babel, opts = {}) {
         }
         const calleePath = path.get('callee');
         switch (calleePath.node.type) {
+          case 'CallExpression':
```

Comment:
By dumping the calleePath object I saw that currying could be handled by handling the 'CallExpression' type at this level. A curried call will create a structure like this: `VariableDeclaration -> CallExpression -> CallExpression -> Identifier`.

So going through the second `CallExpression` allows currying. I did not try deep currying yet, but my guess is that it should work.